googleapi/googleapi_test.go

```diff
@@ -312,6 +312,32 @@ Details:
 		},
 		`googleapi: got HTTP response code 500 with body: {"error":{}}`,
 	},
+	{
+		// Case: Error in JSON object in body.
+		&http.Response{
+			StatusCode: http.StatusTooManyRequests,
+		},
+		`{"error":{"code": 429,"message": "Resource has been exhausted (e.g. check quota).","status": "RESOURCE_EXHAUSTED"}}`,
+		&Error{
+			Code:    http.StatusTooManyRequests,
+			Message: "Resource has been exhausted (e.g. check quota).",
+			Body:    "{\"error\":{\"code\": 429,\"message\": \"Resource has been exhausted (e.g. check quota).\",\"status\": \"RESOURCE_EXHAUSTED\"}}",
```

Comment:
Use backtick so that we don't have to escape these double quotes everywhere.
```suggestion
			Body:    `{"error":{"code": 429,"message": "Resource has been exhausted (e.g. check quota).","status": "RESOURCE_EXHAUSTED"}}`,
```

Reply:
Lol, it was easier to copy in the escaped quotes from console than to use backticks and remove them, so no labor lost here. But I see your point about consistency.